lib/src/spec/common.dart

```diff
@@ -237,39 +225,39 @@ class TokenRequest {
   /// TE4
   int ttl;
 
-  TokenRequest({
-    this.keyName,
-    this.nonce,
-    this.clientId,
-    this.mac,
-    this.capability,
-    this.timestamp,
-    this.ttl
-  });
+  TokenRequest(
+      {this.keyName,
+      this.nonce,
+      this.clientId,
+      this.mac,
+      this.capability,
+      this.timestamp,
+      this.ttl});
 
   /// TE7
-  TokenRequest.fromMap(Map<String, dynamic> map){
-    this.keyName = map["keyName"] as String;
-    this.nonce = map["nonce"] as String;
-    this.mac = map["mac"] as String;
-    this.capability = map["capability"] as String;
-    this.clientId = map["clientId"] as String;
-    this.timestamp = DateTime.fromMillisecondsSinceEpoch(map["timestamp"] as int);
-    this.ttl = map["ttl"] as int;
+  TokenRequest.fromMap(Map<String, dynamic> map) {
+    keyName = map['keyName'] as String;
+    nonce = map['nonce'] as String;
+    mac = map['mac'] as String;
+    capability = map['capability'] as String;
+    clientId = map['clientId'] as String;
+    timestamp = DateTime.fromMillisecondsSinceEpoch(map['timestamp'] as int);
+    ttl = map['ttl'] as int;
   }
-
 }
 
 /// Params for rest history
 ///
 /// https://docs.ably.io/client-lib-development-guide/features/#RSL2b
 class RestHistoryParams {
-  /// start must be equal to or less than end and is unaffected by the request direction
+  /// start must be equal to or less than end and is unaffected
```

Comment:
References to identifiers should be wrapped in `[]` => `/// [start] must ...

Reply:
02f59f4fb8bbe6bc7c68e52e605fb95c42c7e0e5